anyway/flask_app.py

```diff
@@ -1311,6 +1312,29 @@ def get_dict_file(directory):
         yield name, df
 
 
+@app.route("/markers/polygon/", methods=["GET"])
```

Comment:
@erangil  can you add an example of a use in the API?

Reply:
Done